websites.yml

```diff
@@ -144,3 +144,6 @@ avatar:
   tumblr: opengraph
   twitter:
     url: https://twitter.com/{u}/profile_image?size=original
+logos:
+  keybase:
+    url: https://png.icons8.com/ios/30/000000/keybase.png
```

Comment:
Since we cannot get user avatar for Keybase easily, I think we should have their official logo. https://upload.wikimedia.org/wikipedia/commons/b/bb/Keybase_logo_official.svg maybe?

Reply:
Yep that could work too. However, I've submitted an issue for upgrading to font awesome 5, which does include Keybase icons.